src/components/dialogs/replace-with-script-dialog.js

```diff
@@ -48,10 +49,12 @@ const ReplaceWithScriptDialog = ({ id, open, onClose, onClick, title }) => {
                 </DialogContentText>
             </DialogContent>
             <DialogActions>
-                <Button onClick={handleClose} variant="outlined">
-                    <FormattedMessage id="cancel" />
-                </Button>
-                <Button onClick={handleClick}>
+                <CancelButton
+                    onClick={handleClose}
+                    variant="outlined"
+                    withCustomColor={true}
+                />
+                <Button onClick={handleClick} color={'customButton'}>
```

Comment:
Validate in grey too ?

Reply:
Yes, when the cancel button is customized, the secondary button should be white on the dark theme and black on the light theme